iBudget/utils/validators.py

```diff
@@ -98,6 +99,24 @@ def login_validate(data):
     return True
 
 
+def input_spending_registration_validate(data):
+    """validate data.
+        Args:
+            data (dict): contain category, type of pay, sum, comment
+        Returns:
+            bool: The return value. True is data valid, else False.
+    """
+    if not isinstance(data['category'], int):
```

Comment:
There is a chance that data['category'] can be int as a string - e.g. `1`. We can't just reject it. Ideally, we need to check first if it can be converted into int and only then reject. Otherwise, I think, that data type verification is useless in this case.

Reply:
Done